user/zrepl-dsh2dsh/files/tmpfiles.conf

```diff
@@ -0,0 +1,3 @@
+# Directory for control socket
+
+d /run/zrepl 0755 root root -
```

Comment:
it can't create it by itself?

Reply:
No. It fails without that. I'll file that as a bug. The default is /var/run/zrepl/control, but it may be changed in the config.